examples/io/elekta_average.py

```diff
@@ -0,0 +1,63 @@
+"""
```

Comment:
@agramfort that's from older data acquisition system (<3.4), which has somewhat different parameters. I haven't yet implemented support for it.